<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskAllocationModeWindow</class>
 <widget class="QWidget" name="taskAllocationModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Allocation Mode</string>
  </property>
  <widget class="QComboBox" name="modeCbox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Automatic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manual</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Automatic Mode</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Allocation Strategy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="allocationCBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Next Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Best Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Worst Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>First Fit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>First Fit Dec.</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheduling Policy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="schedulingCBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>EDF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RM</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>#CPU</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nCPUEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="simulTimeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="next_run_Button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_run_Button_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
